flask-server/models/forum_models.py

```diff
@@ -0,0 +1,50 @@
+import sys
+sys.path.append("..")
+from exts import db
+from datetime import datetime as dt, UTC
+
+
+class Message(db.Model):
+    """
+    Class model for the Message Board table.
+
+    Attributes
+    ----------
+    post_id : int
+        id of the message
+    post_content : str (text)
+        text content of the message
+    post_category : str
+        category of the message
+    post_author : str
+        author of the message
+    post_date : datetime
+        time and date that the message was posted
```

Comment:
Being nitpicky on these. It's good that you made these, but in their state they are kind of... obvious. What would be interesting to note here is for example that the id is system generated or not. And if it's used as a PK or not.
If there are any unique constraints on them etc.

Reply:
Updated descriptions
[47059db](https://github.com/Angel2001-programmer/Introverse/pull/16/commits/47059dbe82c7564b29c46e503189daaa6c0c9356)